### DISCUSSION TITLE
Dynamic link screen on table rows

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7680] [Help]
is it possible to have a Dynamic link screen on table rows ?
The link I want is stored in a field of my table data provider... but I don't see the table fields in available binding for link screen.
<img width="931" alt="image" src="https://user-images.githubusercontent.com/28592069/188900063-8e9e2c80-da7f-4ead-baa5-82f1e3e32a9a.png">

**Comment by melohagan** [comment]
Hey @NicolasFabre81 

If you need access to the table row bindings, you can nest a link within your table:
![Screenshot 2022-09-13 at 14 58 53](https://user-images.githubusercontent.com/101575380/189920938-515a6fcd-188f-4011-99ef-d900ac244829.png)

The *Link table rows* behaves a little differently in that it automatically picks up the id of the row to use in the link. If you click on the dropdown, then you can see the list of available screens:
![Screenshot 2022-09-13 at 15 00 23](https://user-images.githubusercontent.com/101575380/189921276-898bfec6-ae23-4e03-9b52-1d1681d237c4.png)

In this case Budibase knows to replace `:id` with the id of the row.
